### PR TITLE
Frontend with persistent desktops

### PR DESCRIPTION
**Pull request by jvinolas**
This will get new endpoints on api and backend logic to handle persistent desktops

### CODE REVIEW

**Review by sim6 — CHANGES_REQUESTED** [review]
Endpoint /api/v2/desktops always says "type: persistent" and never shows the template.

**Review by sim6 — DISMISSED** [review]
Now, the endpoint http://localhost:7039/api/v2/user/local-default-admin-admin/desktops always returns to me:
```
"type": "persistent", "template": ""
```
Also the endpoint https://localhost/api/v2/desktops remains like https://github.com/isard-vdi/isard/pull/353#pullrequestreview-555926259

File: backend/isard/desktop.go [hunk 1]
```diff
@@ -94,7 +94,85 @@ func (i *Isard) DesktopDelete(id string) error {
...
 
-		return fmt.Errorf("get user templates: %w", err)
+		return fmt.Errorf("delete desktop: %w", err)
+	}
+
+	return nil
+}
+
+func (i *Isard) DesktopStart(id string) error {
+	req, err := http.NewRequest(http.MethodGet, i.url("desktop/start/"+id), http.NoBody)
```
Comment (sim6) [inline backend/isard/desktop.go:104]: I cannot found this endpoint.

File: backend/isard/desktop.go [hunk 2]
```diff
@@ -94,7 +94,85 @@ func (i *Isard) DesktopDelete(id string) error {
...
+		)
+
+		return fmt.Errorf("start desktop: %w", err)
+	}
+
+	return nil
+}
+
+func (i *Isard) DesktopStop(id string) error {
+	req, err := http.NewRequest(http.MethodGet, i.url("desktop/stop/"+id), http.NoBody)
```
Comment (sim6) [inline backend/isard/desktop.go:143]: I cannot found this endpoint.

**Review by sim6 — COMMENTED** [review]
Now the desktops list is fine. But I can start and stop desktops. I cannot found the isard-api endpoints for that.

**Review by NefixEstrada — APPROVED** [review]

**Comment by fgonzalezmacias** [comment]
Hello

This is a feature that will be great for me in my environment. I have updated and I see the change of look, what I do not see, is where I specify, that a user will be assigned a persistent desktop

image

Thanks in advance

**Comment by sim6** [comment]
We will document it but meanwhile...

Currently you only can start, open viewers and stop persistent desktops via simplified interface.
So you need first create it via full interface. Each user can create their persistent desktops or you as manager (or admin) could go to Manager (Administrator) / Desktops menu and click top right Bulk desktops button to create a persistent desktop for desired users.

**Comment by fgonzalezmacias** [comment]
Ok, Thank you

**Pull request merged**